image/templates/helm/shared/templates/02-scanner-v4-08-indexer-service.yaml

```diff
@@ -0,0 +1,48 @@
+{{- include "srox.init" . -}}
+{{- if ._rox.scannerV4._indexerEnabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: scanner-v4-indexer
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "srox.labels" (list . "service" "scanner-v4-indexer") | nindent 4 }}
+  annotations:
+    {{- include "srox.annotations" (list . "service" "scanner-v4-indexer") | nindent 4 }}
+spec:
+  selector:
+    app: scanner-v4-indexer
+  clusterIP: None
+  ports:
+  - name: grpcs-scanner-v4-indexer
+    port: 8443
+    targetPort: 8443
+  {{- if ._rox.scannerV4.exposeMonitoring }}
+  - name: monitoring
+    port: 9090
```

Comment:
should be the parameterized port I think?

Reply:
changed